src/homepageExperience/utils.ts

```diff
@@ -6,27 +6,27 @@ export const HOMEPAGE_NAVIGATION_STEPS = [
     glyph: IconFont.BookOutline,
   },
   {
-    name: 'Install\nDependencies',
+    name: `Install\n Dependencies`,
```

Comment:
since there's no templating happening in this string (i.e. no `${variable}`) it's better to use single or double quotes (single quotes in our case since that's our standard for strings). Reserving backticks `` ` `` for string interpolation means that when you see a string using `` ` ``, you can be pretty sure it's going to have a variable being output in it.

Reply:
Good catch, I forgot to change that after I tested something when creating the tests!